src/endpoints/utils.js

```diff
@@ -0,0 +1,24 @@
+const log = require('loglevel');
+
+const rapidProUtils = require('../rapidpro-utils');
+
+const sync = async (endpointName, upsert, queryString = '') => {
+  let url = rapidProUtils.getApiUri(endpointName, queryString);
+  let total = 0;
+  while (url) {
+    const result = await rapidProUtils.get(url);
+    const results = result.results;
+
+    log.debug(`fetched ${results.length} from ${endpointName}`);
+    total += results.length;
+
+    await upsert(results);
```

Comment:
This function is called by every endpoint sync, and the error is caught in their specific files, for example: https://github.com/medic/rapidpro2pg/blob/add-more-endpoints/src/endpoints/runs.js#L62

Do you think it would be helpful to catch this error here, just to display an additional message and the propagate the throw to the main catch?